### PR TITLE
Exposes the namespace of the check to the CRD and the metrics endpoint

### PR DESCRIPTION
**Pull request by ihoegen**
Addresses #43

### CODE REVIEW

**Comment by ihoegen** [comment]
Yeah the Component Status Check returns "" for the CheckNamepsace method

File: cmd/kuberhealthy/kuberhealthy.go [hunk 1]
```diff
@@ -390,3 +399,13 @@ func (k *Kuberhealthy) getCurrentState() (health.State, error) {
...
 }
+
+// getCheck returns a Kuberhealthy check object from its name, returns an error otherwise
+func (k *Kuberhealthy) getCheck(name string) (KuberhealthyCheck, error) {
+	for _, c := range k.Checks {
+		if c.Name() == name {
+			return c, nil
+		}
+	}
+	return nil, fmt.Errorf("Could not find Kuberhealthy check with name %s", name)
```
Comment (lolimjake) [inline cmd/kuberhealthy/kuberhealthy.go]: This project uses the logrus package for logging output.

**Comment by integrii** [comment]
The alternative would be to alter the interface's return `[]string` here:

https://github.com/Comcast/kuberhealthy/blob/master/cmd/kuberhealthy/kuberhealthyCheck.go#L35

I'm not sure what is worse - having to implement dead methods, or having to assume some set of return values...

I think I lean toward the way you've done it...

File: cmd/kuberhealthy/kuberhealthy.go (hunk 1 above)
Comment (ihoegen) [reply cmd/kuberhealthy/kuberhealthy.go]: fmt.Errorf doesn't output, it returns an error

**Pull request merged**